schema/changelog-4.17.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd"
+  logicalFilePath="changelog-4.17">
+
+  <changeSet author="author" id="changelog-4.17">
+
+    <renameColumn tableName="tc_orders" oldColumnName="toAddress" newColumnName="toAddressTmp" />
+    <renameColumn tableName="tc_orders" oldColumnName="fromAddress" newColumnName="fromAddressTmp" />
+
+    <addColumn tableName="tc_orders">
```

Comment:
I meant that we should rename existing columns back to "toaddress" and "fromaddress". Instead of adding new ones.

Reply:
Sorry, I don't understand things now, even after sleeping over the matter. I did the rename to lowercase, then you suggested to create temporary tables, then you suggest to rename again.
With the latest changes in master, at least for postgres, you have to have the lowercase columns. If the db engine does not make a distinction between lower and upper case columns, I guess that adding columns with the lowercase names could be problematic. I think the db engines and liquibase will figure out nicely how to do the rename and as you mentioned the orders are probably not used yet. So I suggest to use  7618d84

Reply:
I suggested renaming to temporary names and then renaming to all lower case. That was my second suggestion all along.

> I think the db engines and liquibase will figure out nicely how to do the rename and as you mentioned the orders are probably not used yet.

I'm pretty it doesn't work in some cases. And that's why I suggested not to do direct renaming like you had in the first place.

Reply:
Yes, double rename.

Reply:
check